### PR TITLE
Added Framerate Options

### PR DESCRIPTION
**Pull request by Allstreamer**
This includes:
- Foreground Fps Option
- Background Fps Option
- English Localization
- German Localization

Behavior Changes in this Commit:
On Start of Application Target Framerate is now being set

TARGET_FPS_FOREGROUND & TARGET_FPS_BACKGROUND are now able to be set

### CODE REVIEW

**Comment by Allstreamer** [comment]
I've tried to stick as close as possible to your style and your commit guidelines.

**Review by mbrlabs — CHANGES_REQUESTED** [review]
Thank you, looks good! Just left a comment about the translation.

File: lorien/Assets/I18n/de.txt [hunk 1]
```diff
@@ -59,6 +59,8 @@ SETTINGS_LANGUAGE           Sprache
 SETTINGS_THEME              Theme
 SETTINGS_AA_METHOD          Anti Aliasing Methode
 SETTINGS_AA_METHOD_NONE     Keine
+TARGET_FPS_FOREGROUND       Vordergrund-Bildfrequenz (Fps)
+TARGET_FPS_BACKGROUND       Hintergrund-Bildfrequenz (Fps)
```
[the remark this answers is not in the document]
Comment (Allstreamer) [reply lorien/Assets/I18n/de.txt:63]: Ja, hast recht

**Review by mbrlabs — APPROVED** [review]
Thanks!

**Pull request merged**